elixir/apps/web/lib/web/auth.ex

```diff
@@ -640,4 +672,8 @@ defmodule Web.Auth do
 
     real_ip || peer_data.address
   end
+
+  defp put_client_auth_cookie(conn, state) do
```

Comment:
Please put it next to `put_client_auth_state_from_cookie`, it's better to keep a function that writes something next to the one that reads it.